entries/entry05.md

```diff
@@ -1,7 +1,50 @@
-# Entry 5
-##### X/X/XX
+# Entry 5: Blog entrys are a pain
```

Comment:
Sources: 0 (your sources need to be linked!)
EDP: 1
Knowledge: 1 (consider adding some screenshots in the future!)
Skills: 0
Professionalism: 1

Great job!